site/content/entry/citra-progress-report-2019-q1.md

```diff
@@ -7,156 +7,156 @@ coauthor = "jroweboy"
 forum = 96286
 +++
 
-Hey there Citra fans! These past 6 months have been crazily exciting, and simply wonderful for Citra overall.
+Hey there, Citra fans! These past 6 months have been crazily exciting, and simply wonderful for Citra overall.
 We've made major breakthroughs which were long overdue.
 New features, bug fixes, performance improvements, improved game compatibility, and much more await you.
-So, buckle up and enjoy the ride!
+So buckle up and enjoy the ride!
 <!--more-->
 ****
-To start off - Back in August, we asked our patrons to vote on what they'd like to see us work on the most and now it's time to review how much progress we've made on everything since then.
+To start off: back in August, we asked our patrons to vote on what they'd like to see us work on the most, and now it's time to review how much progress we've made on everything since then.
```

Comment:
change the `:` to a `,` https://www.grammarbook.com/punctuation/colons.asp

Reply:
All right.